### ISSUE TITLE
Gaps between squares in SVG

### ISSUE THREAD

**Issue by andrewbaxter** [erwanvivien/fast_qr#67]
Hello, looking at the generated in chromium sometimes I could see gaps between the squares (in the corners?).

![image](https://github.com/user-attachments/assets/b139b1e8-4270-404d-9126-9d9af2a9be8f)
![image](https://github.com/user-attachments/assets/3f526264-9d63-4976-9173-7f0ab86e0046)

By the way, this is a great library, thanks for working on this!

**Comment by erwanvivien** [comment]
Hey 👋 Thanks for reporting this, it seems weird to me 🤔 
It might be due to anti-aliasing or rounding errors, I'll have to check but don't have much time right now...

**Comment by andrewbaxter** [comment]
Would it be possible to aggregate squares sharing sides into a single filled polygon?

**Comment by erwanvivien** [comment]
Yep in theory it is, but the library doesn't work like that... You can have other shapes than squares that's the problem
